pkg/ui/workspaces/cluster-ui/src/statementDetails/statementDetails.tsx

```diff
@@ -503,6 +503,9 @@ export class StatementDetails extends React.Component<
       moment(stats.last_exec_timestamp.seconds.low * 1e3).format(
         "MMM DD, YYYY HH:MM",
       );
+    const notSelectStatement =
```

Comment:
https://github.com/cockroachdb/cockroach/issues/67084 
The ticket has been updated to not show "mean rows written" for SELECT statements. Is there a better way to check for SELECT statements? I'm not sure if this would always work